dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/WebsiteRepositoryImpl.java

```diff
@@ -67,13 +66,9 @@ public static String getSqlUpdateFieldValues() {
   private final WebpageRepositoryImpl webpageRepositoryImpl;
 
   @Autowired
-  public WebsiteRepositoryImpl(
-      Jdbi dbi,
-      IdentifierRepository identifierRepository,
-      WebpageRepositoryImpl webpageRepositoryImpl) {
+  public WebsiteRepositoryImpl(Jdbi dbi, WebpageRepositoryImpl webpageRepositoryImpl) {
```

Comment:
Hier wird nach wie vor ein Repository verwendet, gehört wohl eher in den Service, oder?

Reply:
Wird nur für Select gebraucht. Könnte also hier bleiben.

Reply:
Ok :+1: